src/main/java/com/onlinebookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -51,4 +51,13 @@ protected ResponseEntity<Object> entityNotFoundException(EntityNotFoundException
         body.put("message", ex.getMessage());
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler(RegistrationException.class)
+    protected ResponseEntity<Object> registrationException(RegistrationException ex) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", LocalDateTime.now());
+        body.put("status", HttpStatus.BAD_REQUEST);
+        body.put("message", ex.getMessage());
+        return new ResponseEntity<>(body, HttpStatus.BAD_REQUEST);
```

Comment:
I see similar logic in the previous method. Let’s move it to the private method ;)